src/services/production.ts

```diff
@@ -223,7 +220,6 @@ export const services: ServiceConfigurator<SupportedTokens> = ($) => [
     service: useBootstrap,
     arguments: [
       $.logger,
-      $.api,
```

Comment:
Why is this change needed here?

Reply:
Because the bootstrap function no longer uses the API service and so doesn’t need it. The reason it was using it was that we initialized the logger with the result to an API call to `/config` which we no longer do with this change.